.github/workflows/on-push.yml

```diff
@@ -0,0 +1,24 @@
+name: push-event
+
+on:
+  push:
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out github-actions repo
+        uses: actions/checkout@v4
+        with:
+          repository: aerius/github-actions
+          path: aerius-github-actions
+          ref: v1.1
+
+      - uses: aerius/github-actions/events/push-event-action@v1.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
If this secret doesn't exist (which is the case at the moment), it will just skip the Sonar step. Is that what we want or do we want to set that up directly as well?

Reply:
If it skips the sonar step I guess it's fine to leave it in :ok_hand: I personally don't really care about sonar or code quality in this project, because it's just a code generator project, and the quality of the generated code is what actually matters rather than the veritable garbage that is the generator.

When eventually we do want some sanity checks or basic quality checks in here or whatever, we can set it up then